pom.xml

```diff
@@ -73,8 +73,40 @@
         <dockerProjectVersion>${dockerVersionSeperator}${project.version}</dockerProjectVersion>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging-log4j2</artifactId>
+            <version>1.0.0-SCMOD-7476-SNAPSHOT</version>
+        </dependency>
+    </dependencies>
+
     <build>
         <plugins>
+            <!-- Unpack the base worker configuration files -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>process-config-files</id>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.github.cafapi</groupId>
+                                    <artifactId>caf-logging-log4j2</artifactId>
+                                    <outputDirectory>${project.build.directory}/config-files</outputDirectory>
+                                    <includes>log4j2.*</includes>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
I don't understand why these are being extracted from the jar.

Reply:
Included log4j.properties with image instead of extracting from jar